### DISCUSSION TITLE
MSSQL errors [Jul 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2858] [Discord Threads]
> 🙋‍♂️ **00Cedge**  
#### Thanks!!  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/998834306761887895/unknown.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Could you please raise an issue - include as much details as possible  
#  
> 🙋‍♂️ **00Cedge**  
#### Hi <@310157751395287040> thanks for your reply.  Sorry how do I raise an issue?  Can I DM you with connection details etc?  
#  
> 💁 **o1lab**  
#### ya sure, please do  
#  
> 💁 **o1lab**  
#### happy to take a look  
#  
> 💁 **o1lab**  
#### I was mostly indicating to raise an issue in our github  
#  
> 💁 **o1lab**  
#### 😄  
#  
> 💁 **o1lab**  
#### by redacting credentials  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/998834307705614377)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
